test/e2e/tests/phishing-detection.spec.js

```diff
@@ -36,15 +36,20 @@ describe('Phishing Detection', function () {
         ganacheOptions,
         title: this.test.title,
         testSpecificMock: mockPhishingDetection,
+        dapp: true,
       },
       async ({ driver }) => {
         await driver.navigate();
         await driver.fill('#password', 'correct horse battery staple');
         await driver.press('#password', driver.Key.ENTER);
-        await driver.openNewPage('http://example.com');
-        await driver.waitForSelector({ text: 'continuing at your own risk' });
+        await driver.navigate();
+        await driver.openNewPage('http://localhost:8080');
+        const continueLink = await driver.waitForSelector({
+          text: 'continuing at your own risk',
+        });
+        await continueLink.click();
```

Comment:
Nit: This seems simpler

```suggestion
        await driver.clickElement({
          text: 'continuing at your own risk',
        });
```

The `waitForSelector` method is for when you _just_ want to wait. All of the "interact" functions have built-in functionality to wait until the default timeout until the selector is found.